applications/ATOM/data_config.json

```diff
@@ -1,7 +1,5 @@
 {
-
-  "pad_index": 28,
-  "max_seq_len": 57,
-  "data_path": "/p/lustre2/brainusr/datasets/zinc/moses_zinc_train250K.npy"
-
+  "pad_index": 34,
```

Comment:
Please reverse the changes here, lets keep the default value.

Reply:
fixed.